docs/mpcnc/Pbase.md

```diff
@@ -0,0 +1,148 @@
+
+![!Compare measurements](https://www.v1engineering.com/wp-content/uploads/2020/06/base-check-2-scaled.jpg){: width="400"}
+
+* This next diagonal should match the previous one. Squaring this way can be extremely accurate as the deviation is multiplied.
+* Adjust both rear legs until you are satified with the precision.
+* Screw in one foot at a time, minor adjustments are possible.
+* Triple check the diagonals they need to be as close as you can get them.
+
+!!! Tip
+    Marking and pre-drilling the foot hold downs in the center of the screw slots makes it much easier to get a square base and leaves a tiny bit of room for adjustment.
+
+___
+
+## Leg Locks
+![!Parts and hardware](https://www.v1engineering.com/wp-content/uploads/2020/06/lock-parts-scaled.jpg){: width="400"}
+
+* You will need the Printed Corner Leg Locks and M5 Luck nuts.
+* It is best to pre-thread all the locknuts before assembly. This will loosen them and prevent mangaling of the printed parts.
+
+![!Embeded nut](https://www.v1engineering.com/wp-content/uploads/2020/06/lock-ready-scaled.jpg){: width="400"}
+
+* Push the nut into the Leg lock and fully seat it. If it does not stay on it's own you can use a drop of Glue.
+* A second option if it does not stay on it's own is to start assembling the Corner Bottom very loosely.
+
+![!Orientation](https://www.v1engineering.com/wp-content/uploads/2020/06/base-lock-scaled.jpg){: width="400"}
+
+* Fully seat the Corner Leg Locks onto the legs.
+* Orient to match the Feet below.
+* Do not lock/clamp them.
+
+___
+
+## Corner Bottoms
+![!Parts and hardware](https://www.v1engineering.com/wp-content/uploads/2020/06/corner-bottom-scaled.jpg){: width="400"}
+
+* You will need the printed Corner Bottoms and Corner Bottom Mirrored And M5x30 screws.
+* If you are not using the V1 Kit please make sure your screw heads are low profile enough not to interfere with the Lower Rail.
+
+![!Belt Orientation](https://www.v1engineering.com/wp-content/uploads/2020/06/base-bottom-arrows-scaled.jpg){: width="400"}
+
+* The corner bottoms go on the leg locks as shown, Pay attention to how the belt slider slots are oriented. They point towards each other on the outside of the build area.
+* This picture is taken with the 0,0 (home) corner in the bottom nearest left. The corners should be as shown.
+
+![!Assemble](https://www.v1engineering.com/wp-content/uploads/2020/06/base-bottom-scaled.jpg){: width="400"}
+
+* Get the screws started to hold the leg locks and corner bottoms together.
+* Leave them loose, just tight enough to make the screw head clear the rail.
+
+![!Add Y rails](https://www.v1engineering.com/wp-content/uploads/2020/06/rails-1-scaled.jpg){: width="400"}
+
+* Temporarily adding the Y rails will align the corners.
+
+___
+
+## Lock The Legs
+![!Hardware needded](https://www.v1engineering.com/wp-content/uploads/2020/06/leglock1-scaled.jpg){: width="400"}
+
+* You will need four sets of the M5 screws and Locknuts.
```

Comment:
If you wanted to be very careful, and always do a little "parts you need" at the top of each step, a table might work pretty  well. Something like:

| Part | Qty |
|---|---|
| M5x25mm Screw | 4 |
| M5 Locknut | 4 |

Maybe that should be in round 2, or some refinement for "later"